doc/man7/ssl.pod

```diff
@@ -107,679 +107,6 @@ it's already included by ssl.h>.
 
 =back
 
-=head1 API FUNCTIONS
-
-Currently the OpenSSL B<ssl> library exports 214 API functions.
-They are documented in the following:
-
-=head2 Dealing with Protocol Methods
```

Comment:
While I agree with the removal of inaccurate information, or for that matter a "complete" run-down of the API in a man7 page, I do not agree with  removing the guidance completely.  Of course, there's that SEE ALSO list, but quite frankly, with that Big Wall of Links, my personal reaction would be to thrown my hands in the air and admit defeat...

So I propose some kind of guidance still, maybe roughly like this:

``` pod
=head1 TOPICS

=head2 Dealing with Protocol Methods

There are a few TLS protocols to choose from, do see L<SSL_...(3)> and L<SSL_...(3)> as a starting point.

=head2 Dealing with Ciphers

.... yada yada a couple of links to starting points yada yada...

=head2 Dealing with Protocol Contexts

... et ...

=head2 Dealing with Sessions

... cetera ...

=head2 Dealing with Connections

...

```

How does that sound?

Reply:
> So I propose some kind of guidance still, maybe roughly like this:

I think that sounds great, if someone else wants to do it.  I just wanted to get the outdated stuff gone.

Reply:
I think I'll wait for a PR that doesn't slash and leave the user helpless, but replaces it with something sensible.  Thank you.

Reply:
We differ on whether the current page is helpful or not. :)  I'll wait a bit and see if anyone else on the project likes this before closing it.

Reply:
If this page isn't useful at all, why not simply removing it? So I guess that my question, do you think that a guide in this form may be useful at all or not? If you think it is, why not make it so?